finetune/config.py

```diff
@@ -232,24 +237,22 @@ def get_default_config():
         shuffle_buffer_size=100,
         dataset_size=None,
         batch_size=2,
-        visible_gpus=None, # defaults to all available
+        visible_gpus=None,  # defaults to all available
         n_epochs=GridSearchable(3, [1, 2, 3, 4]),
         seed=42,
         max_length=512,
         weight_stddev=0.02,
         save_dtype=None,
         val_set=None,
         per_process_gpu_memory_fraction=0.95,
-        adapter_size = None, #from Parameter Efficient Transfer Learning paper
-
+        adapter_size=None,  # from Parameter Efficient Transfer Learning paper
         # Regularization
         embed_p_drop=0.1,
         attn_p_drop=0.1,
         resid_p_drop=0.1,
         clf_p_drop=0.1,
         l2_reg=GridSearchable(0.01, [0.0, 0.1, 0.01, 0.001]),
         vector_l2=False,
-
```

Comment:
These spaces, whilst not conforming to formatting standards greatly improve readability, so unless anyone has a large problem id vote to put them back in.